patches/minecraft/net/minecraft/entity/ai/attributes/GlobalEntityTypeAttributes.java.patch

```diff
@@ -0,0 +1,23 @@
++   private static final Map<EntityType<? extends LivingEntity>, AttributeModifierMap> FORGE_ATTRIBUTES = new java.util.HashMap<EntityType<? extends LivingEntity>, AttributeModifierMap>();
++   
++   public static AttributeModifierMap put(EntityType<? extends LivingEntity> type, AttributeModifierMap map) {
++       return FORGE_ATTRIBUTES.put(type, map);
```

Comment:
Is it worth adding a sanity check to prevent copypasta calling put() twice on the same EntityType?

Reply:
There could be a case someone may want to override another mod's attribute map.

Reply:
In that case, would we want to similarly allow adding properties to vanilla entities by favoring forge's output and using vanilla as a fallback?

Reply:
Makes sense to me. Although if we were to do that it would probably require more in-depth features such as the ability to register on top of existing maps (they're builder based). This PR is mainly a quick addition so modders can actually USE attributes

Reply:
Upon further inspection, it seems the maps have a bunch of public methods that allow this. So vanilla and modded can be modified. Although it appears some can be non mutable, so favoring Forge makes more sense. I'll do that.